QUALITY_DECLARATION.md

```diff
@@ -1,132 +1,194 @@
 
 There is an automated test which runs a linter that ensures each file has a license statement.
 
-### Copyright Statements
+### Copyright Statements [3.iv]
 
 The copyright holders each provide a statement of copyright in each source code file in `rcutils`.
 
 There is an automated test which runs a linter that ensures each file has at least one copyright statement.
 
-## Testing
+## Testing [4]
 
-### Feature Testing
+### Feature Testing [4.i]
 
-Each feature in `rcutils` has corresponding tests which simulate typical usage, and they are located in the `test` directory.
+Each feature in `rcutils` has corresponding tests which simulate typical usage, and they are located in the [`test`](https://github.com/ros2/rcutils/tree/master/test) directory.
 New features are required to have tests before being added.
 
-### Public API Testing
+### Public API Testing [4.ii]
 
-Each part of the public API have tests, and new additions or changes to the public API require tests before being added.
+Each part of the public API has tests, and new additions or changes to the public API require tests before being added.
 The tests aim to cover both typical usage and corner cases, but are quantified by contributing to code coverage.
 
-### Coverage
-
-TODO fix link
+### Coverage [4.iii]
 
-`rcutils` follows the recommendations for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#coverage), and opts to use branch coverage instead of line coverage.
+`rcutils` follows the recommendations for ROS Core packages in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#code-coverage), and opts to use line coverage instead of branch coverage.
 
 This includes:
 
-- tracking and reporting branch coverage statistics
-- achieving and maintaining branch coverage at or above 95%
+- tracking and reporting line coverage statistics
+- achieving and maintaining a reasonable branch line coverage (90-100%)
 - no lines are manually skipped in coverage calculations
 
 Changes are required to make a best effort to keep or increase coverage before being accepted, but decreases are allowed if properly justified and accepted by reviewers.
 
-Current coverage statistics can be viewed here:
+Current coverage statistics can be viewed [here](https://ci.ros2.org/job/ci_linux_coverage/85/cobertura/src_ros2_rcutils_src/)
 
-TODO FIXME
+### Performance [4.iv]
 
-### Performance
+`rcutils` follows the recommendations for performance testing of C code in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#performance), and opts to do performance analysis on each release rather than each change.
+Soon to be defined if this package requires performance testing and how addresses this topic.
 
-TODO fix link
+### Linters and Static Analysis [4.v]
```

Comment:
again links to the night builds?

Reply:
Added in [efce310](https://github.com/ros2/rcutils/pull/240/commits/efce310a2f85e5b473acb73c29633821fa7d6bd2)